src/bio.c

```diff
@@ -134,7 +134,7 @@ static int wolfSSL_BIO_SSL_read(WOLFSSL_BIO* bio, void* buf,
     WOLFSSL_ENTER("wolfSSL_BIO_SSL_write");
```

Comment:
Can you change this WOLFSSL_ENTER  message to `wolfSSL_BIO_SSL_read`?

Reply:
Yes.